File: j2pBindLib/src/test/resources/SampleCode/AcceptAndReturnTest.py
import loader
import sys
loader.load_library()

from pybind.test.AcceptAndReturnTest import AcceptAndReturnTest
from jnius import autoclass

if __name__ == "__main__":
    t = AcceptAndReturnTest()
    print("Python: Passing string 'Hi'")
    sys.stdout.flush()
    JString = autoclass('java.lang.String')
    t.accept(JString("Hi"))
    print("Accept completed")

File: j2pBindLib/src/test/resources/SampleCode/AcceptAndReturnTest.java
package test;
public class AcceptAndReturnTest {

    public Object accept(Object m){
        System.out.println("Java: Accepted: " + m.getClass().getCanonicalName() + " : " + m.toString());
        System.out.println("Java: Returning input back to caller");
        return m;
    }

    public int acceptInt(int m){
        System.out.println("Java: int accepted: " + m);
        System.out.println("Java: Returning input back to caller");
        return m*2;
    }
}